src/win/winapi.c

```diff
@@ -123,6 +126,14 @@ void uv__winapi_init(void) {
       GetProcAddress(powrprof_module, "PowerRegisterSuspendResumeNotification");
   }
 
+  kernelbase_module = GetModuleHandleA("kernelbase.dll");
```

Comment:
Windows now does API versioning by giving the dynamic library a weird (fake, internal) name, in this case it is:
api-ms-win-core-processthreads-l1-1-3.dll

From: https://learn.microsoft.com/en-us/uwp/win32-and-com/win32-apis